common-requests/use.md

```diff
@@ -1,8 +1,157 @@
 ---
 layout: page
-title: "Usage"
+title: "How to Use Open Horizon"
 description: "TBD"
 ---
 
-# Use
+# How to Use Open Horizon
 
+## Now that I have it installed, what next?
+
+Let's start by getting familiar with the CLI (Command Line Interface), 
+which is the primary way you can interact directly with the Anax Agent and the Management Hub services.  The CLI uses a short, easy-to-remember synonym for the word "horizon", `hzn`.  If you remember nothing else about how to use Open Horizon, just type in `hzn` from the command prompt to get a complete view of how to get started using the CLI.
+
+## Your first command
+
+The first command to try can be the simple "version", which will tell you the version numbers of both the CLI and the Agent.  If the CLI can't tell you the Agent version, or it doesn't match the CLI version, you have a problem.  This will most likely not happen.  Give it a try:
+
+``` shell
+hzn version
+```
+
+At the time we wrote the documentation, that command returned this:
+
+``` text
+Horizon CLI version: 2.26.12
+Horizon Agent version: 2.26.12
+```
+
+## Get the Agent's configuration
+
+Now that we've established that the CLI is working, let's use the "node" command to get the node's configuration from the Agent:
+
+``` shell
+hzn node list
+```
+
+Notice that the typical pattern we're following here is [CLI] [Object Noun] [Action Verb]. 
+The response that you will get back will look vaguely like this JSON example:
+
+``` json
+{
+  "id": "node1",
+  "organization": "myorg",
+  "pattern": "",
+  "name": "node1",
+  "nodeType": "device",
+  "token_last_valid_time": "2020-08-04 16:00:30 -0400 EDT",
+  "token_valid": true,
+  "ha": false,
+  "configstate": {
+    "state": "configured",
+    "last_update_time": "2020-08-04 16:00:31 -0400 EDT"
+  },
+  "configuration": {
+    "exchange_api": "http://host.docker.internal:3090/v1/",
+    "exchange_version": "2.37.0",
+    "required_minimum_exchange_version": "2.23.0",
+    "preferred_exchange_version": "2.23.0",
+    "mms_api": "http://host.docker.internal:9443",
+    "architecture": "amd64",
+    "horizon_version": "2.26.12"
+  }
+}
+```
+
+This output will require a little more explanation. 
+Beginning with the first two lines, `id` refers to the ID we assigned to this edge node, and `organization` refers to the ad-hoc group that the device "belongs" to. 
+
+Next, take a look at the `configstate.state` property.  If the value is "configured", then this tells you that the edge node is currently registered for a service.
+
+Last, look at the `configuration.exchange_version` property.  If you do not see a version number as the value, this is most likely because the Agent cannot query the Exchange service for the version, which may mean that your Agent is not connected to the Management Hub.  This will most likely not happen using the all-in-one installation.
```

Comment:
Lastly, if you do not see a version number value in the `configuration.exchange_version` property, this is probably because the agent cannot query the exchange service for the version. This might indicate that your agent is not connected to the management hub, but this is not typical in all-in-one installations.